app/services/proofing/aamva/response/verification_response.rb

```diff
@@ -76,13 +84,15 @@ def handle_soap_error
           error_handler = SoapErrorHandler.new(http_response)
           return unless error_handler.error_present?
 
-          msg = error_handler.error_message
-          raise ::Proofing::TimeoutError, msg if mva_timeout?(msg)
-          raise VerificationError, msg
+          @errors.push(error_handler.error_message)
         end
 
         def node_for_match_indicator(match_indicator_name)
-          REXML::XPath.first(rexml_document, "//#{match_indicator_name}")
+          begin
+            REXML::XPath.first(rexml_document, "//#{match_indicator_name}")
+          rescue REXML::ParseException
+            nil
+          end
         end
```

Comment:
I think we could take advantage of the methid-level implicit begin and remove a few keywords:
```suggestion
          REXML::XPath.first(rexml_document, "//#{match_indicator_name}")
        rescue REXML::ParseException
          nil
        end
```

Reply:
Changed.